src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveSummaryService.kt

```diff
@@ -146,11 +146,7 @@ class IncentiveSummaryService(
       }
 
   private fun calcTypeCount(caseNoteUsage: List<CaseNoteUsage>): Int =
-    if (caseNoteUsage.isNotEmpty()) {
-      caseNoteUsage.map { it.numCaseNotes }.reduce { acc, next -> acc + next }
-    } else {
-      0
-    }
+    caseNoteUsage.map { it.numCaseNotes }.fold(0) { acc, next -> acc + next }
```

Comment:
Neat!

Reply:
yeah, I prefer this because you don't have to deal with the empty list case as an exceptional one, works nice as `0` is the identity of the sum ☺️